### PR TITLE
Brainfuck: Added description to brainfuck

### PR DESCRIPTION
**Pull request by navneetzz**
Added missing description to brainfuck cheat sheet

https://duck.co/ia/view/brainfuck_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Brainfuck Cheat Sheet](https://duck.co/ia/view/brainfuck_cheat_sheet)

**Description**: Cheat Sheet for Brainfuck

**Example Query**: [brainfuck cheatsheet](https://beta.duckduckgo.com/?q=brainfuck cheatsheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: http://www.muppetlabs.com/~breadbox/bf/

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/brainfuck_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/brainfuck_cheat_sheet)*

File: share/goodie/cheat_sheets/json/brainfuck.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "brainfuck_cheat_sheet",
     "name": "Brainfuck",
-
+    "description": "List of Brainfuck commands, an esoteric programming language",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/brainfuck.json]: How about `Commands for Brainfuck, an esoteric programming language`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=brainfuck cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/brainfuck_cheat_sheet_mobile.png)

**Comment by GuiltyDolphin** [comment]
@moollaza LGTM :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=brainfuck cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/brainfuck_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=brainfuck cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/brainfuck_cheat_sheet_mobile.png)

**Pull request merged**